README.md

```diff
@@ -184,31 +184,38 @@ Inference data with groups:
 	> observed_data
 ```
 
-The forecast `idata` is accessed via:
+The forecast `idata`s are accessed via:
 
 ```python
 import forecasttools
 
 
-idata = forecasttools.nhsn_flu_forecast
+# idata with dates as coordinates
+idata_w_dates = forecasttools.nhsn_flu_forecast_w_dates
+
+# idata without dates as coordinates
+idata_wo_dates = forecasttools.nhsn_flu_forecast_wo_dates
 ```
 
 The forecast was generated following the creation of `nhsn_hosp_flu.csv` (see previous section) by running `data.py` with the following added:
```

Comment:
I don't think I am seeing where you generate the idata with dates. Where is that?

Reply:
These are the cached influenza forecasts that come with forecasttools. 

There is no current vignette that shows the conversion of an `idata` without dates to an `idata` with dates.

There is an experimentation file that contains this conversion. 

I can convert this experimentation file into a vignette if this is desired.

The contents of the experimentation file:

```python
"""
Experimentation file for
examining how idata with
dates operate.
"""

# %% IMPORT LIBRARIES


import xarray as xr

import forecasttools

xr.set_options(
    display_expand_data=False,
    display_expand_attrs=False,
)


# %% GET IDATA WITHOUT DATES

idata_wo_dates = forecasttools.nhsn_flu_forecast_wo_dates

print(idata_wo_dates)

print(idata_wo_dates.observed_data["obs_dim_0"].values[:10])

# %% ADD DATES TO IDATA OBJECT GROUP(S)


start_date_iso = "2022-08-08"
idata_w_dates = forecasttools.add_dates_as_coords_to_idata(
    idata_wo_dates=idata_wo_dates,
    group_dim_dict={
        "observed_data": "obs_dim_0",
        "posterior_predictive": "obs_dim_0",
        "prior_predictive": "obs_dim_0",
    },
    start_date_iso=start_date_iso,
)

print(idata_w_dates)

print(idata_w_dates.observed_data["obs_dim_0"].values[:10])

print(type(idata_w_dates.observed_data["obs_dim_0"].values[0]))


# %% CONVERT DATES IDATA TO DATAFRAME

df_from_idata_w_dates = forecasttools.idata_forecast_w_dates_to_df(
    idata_w_dates=idata_w_dates,
    location="TX",
    postp_val_name="obs",
    postp_dim_name="obs_dim_0",
    timepoint_col_name="date",
    value_col_name="hosp",
)
df_from_idata_w_dates
```

Reply:
You don't need to do a full-blown vignette, but it would be good if all of the objects used in the library were reproducible.

Reply:
I think "reproduce objects in the package" scripts can be minimally-documented notebooks/vignettes.